dbt_copilot_helper/commands/bootstrap.py

```diff
@@ -66,6 +68,10 @@ def make_config(directory="."):
 
     click.echo(">>> Generating Copilot configuration files\n")
 
+    # add .copilot-helper-version file
+    copilot_version = string_version(get_copilot_versions()[0])
```

Comment:
This will be the copilot version rather than the copilot-helper version

Reply:
@JohnStainsby right, good spot. I'll switch to `get_app_version()` instead

Reply:
Do you know what format `get_app_version` will return the version in? Just had a quick look and I don't think it's x.y.z

Reply:
It uses `parse_version` under the hood, so it'll be a tuple `(local_version, released_version)` of tuples in the `(x,y,z)` format, so it can be then denormalised with `string_version`

Reply:
Ah I see, missed the string_version